datafusion/expr/src/expr_schema.rs

```diff
@@ -328,10 +328,45 @@ impl ExprSchemable for Expr {
                     Ok(true)
                 }
             }
+            Expr::WindowFunction(WindowFunction { fun, .. }) => {
```

Comment:
Is this change required for this PR or is it a  "drive by" improvement?

Reply:
Required